spark2-test-lib/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we can do this without adding a new package. We can add this to test-lib and re-use the Spark version depending build rule. What do you think?